apps/science/src/main.cpp

```diff
@@ -1,149 +1,242 @@
 #include "AdafruitSTEMMA.h"
-#include "CANBus.h"
+#include "CANConfig.h"
 #include "CANMsg.h"
 #include "CentrifugeConfig.h"
 #include "CoverConfig.h"
 #include "DiggerConfig.h"
 #include "ElevatorConfig.h"
 #include "Logger.h"
 
-// TODO: Handle function call failures better
+Sensor::AdafruitSTEMMA moistureSensor(TEMP_MOIST_I2C_SDA, TEMP_MOIST_I2C_SCL);
 
-// Felix TODO: Brose through logic and make sure there aren't any silly mistakes.
+CANInterface can(CANConfig::config);
```

Comment:
I also recommend not making these global. Global variables may lead to unexpected behavior and it's better to pass the sensor and the can interface references to the functions where they are needed imo